internal/convert/process_config.go

```diff
@@ -485,6 +485,7 @@ func (p *ProcessConfig) process() *opsmngr.Process {
 	process := &opsmngr.Process{
 		AuthSchemaVersion:                authSchemaVersion,
 		BackupRestoreCheckpointTimestamp: p.BackupRestoreCheckpointTimestamp,
+		DefaultRWConcern:                 p.defaultRW(),
```

Comment:
is this the main fix? Could you explain why this is needed? I see that `&opsmngr.Process` has many other properties where we don't set the default value and I am not sure why here is needed.

Reply:
Side-effect observed rather than the main change.

By default, the DefaultRWConcern isn't specified and can be safely ignored - it only becomes an issue if the user has updated the setting at some point. Once the value has been set once, it needs to be present in all further config updates, otherwise the config is rejected as the existing value would be changed to nil.  

Regarding the other settings, they would all need to be tested individually to see the impact. Currently, the mongocli cluster update command only covers a subset of allowed settings.